bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -56,6 +56,42 @@
 
 	</thing-type>
 
+	<thing-type id="WHR930">
+		<label>Zehnder WHR930 Ventilation System</label>
+		<description>Provides a generic access to a Zehnder WHR930 ventilation device</description>
+
+		<channel-groups>
+			<channel-group id="bindingControl" typeId="bindingControl"/>
+			<channel-group id="ventilation" typeId="ventilation_WHR930"/>
+			<channel-group id="temperatures" typeId="temperatures_WHR930"/>
+			<channel-group id="times" typeId="times_WHR930"/>
+			<channel-group id="bypass" typeId="bypass"/>
+			<channel-group id="preheater" typeId="preheater"/>
+			<channel-group id="options" typeId="options_WHR930"/>
+			<channel-group id="menuP1" typeId="menuP1_WHR930"/>
+			<channel-group id="menuP2" typeId="menuP2_WHR930"/>
+			<channel-group id="menuP9" typeId="menuP9_WHR930"/>
+			<channel-group id="inputs" typeId="inputs"/>
+			<channel-group id="analogRF" typeId="analogRF"/>
+			<channel-group id="errors" typeId="errors"/>
+			<channel-group id="resets" typeId="resets"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="serialPort" type="text" required="true">
+				<label>Serial Port</label>
```

Comment:
You could move the config into its own tag to need to specify it only once. See here for example: https://github.com/openhab/openhab-addons/blob/66a540289d6e10beb0523db165657c2d298c24ac/bundles/org.openhab.binding.lcn/src/main/resources/OH-INF/config/config.xml#L8

Then, you can add a reference to both things to the config here like here https://github.com/openhab/openhab-addons/blob/66a540289d6e10beb0523db165657c2d298c24ac/bundles/org.openhab.binding.lcn/src/main/resources/OH-INF/thing/thing-types.xml#L10